plugins/rsyslog.yaml

```diff
@@ -77,6 +117,11 @@ pipeline:
     labels:
       log_type: rsyslog
       plugin_id: {{ .id }}
+    tls:
+      enable: {{ .tls_enable }}
+      certificate: {{ .tls_certificate }}
+      private_key: {{ .tls_private_key }}
+      min_version: {{ .tls_min_version }}
```

Comment:
My initial thought was "this should be wrapped with an `if` block but I suppose it does not need to be. It looks clean this way.